include/psa/crypto_values.h

```diff
@@ -41,6 +46,11 @@
 
 /* PSA error codes */
 
+/* Error codes are standardized across PSA domains (framework, crypto, storage,
+ * etc.). Do not change the values in this section. If you must add a new
```

Comment:
Does this affect the storage format at all?

Reply:
No: error codes are not stored in persistent storage. However the PSA standardization makes them even harder to change in practice: we'd have to coordinate over all PSA services. And we must preserve the macro expansions and not just the numerical values. I've added a note to that effect.

Reply:
I've also added a test to ensure we don't accidentally become incompatible with other PSA headers that define status codes. Not in this PR so as not to delay it. https://github.com/Mbed-TLS/mbedtls/pull/5998